cmd/validator/accounts/backup.go

```diff
@@ -16,7 +16,7 @@ import (
 const backupPromptText = "Enter the directory where your backup.zip file will be written to"
 
 func accountsBackup(c *cli.Context) error {
-	w, km, err := walletWithKeymanager(c)
+	w, km, err := walletWithKeymanager(c, nil)
```

Comment:
Feels a bit weird to pass `nil` into a function call like this, what is the purpose?

Reply:
I agree I was trying to think of a better way to clean this up but just had a hard time figuring out how I can pass the genesis validators root here.

at first, I had the node client abstracted away in the function. so that if web3signer was used it would make an API call to get the genesis information but I had a hard time unit testing it, but the more I think of it maybe I can just abstract the logic out into its own function, and not have the unit test  run the connection...

I also tried doing something like `walletWithKeymanager(c *cli.Context, conns ...*grpc.Connection)`but had a hard time mocking the genesis call. open for other suggestions.

Reply:
instead of using the connection i could also hardcode a map like potuz suggested and get it from the cli, but i was thinking other chains that use prysm like gnosis might want this capability too without having exclusive support